controllers/provisioners/eks/update.go

```diff
@@ -64,6 +64,14 @@ func (ctx *EksInstanceGroupContext) Update() error {
 	}
 	instanceProfile := state.GetInstanceProfile()
 
+	if configuration.Image == "latest" {
```

Comment:
Seems this code is repetative in `update.go` and `create.go`, should we simply do this check in `cloud.go` and overwrite the field internally with the resolved value?